recipes/dolfinx_mpc/yum_requirements.txt

```diff
@@ -0,0 +1 @@
+mesa-libGL
```

Comment:
note: this is only needed for `gmsh` in the test env. dolfinx_mpc never uses it directly.